sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -722,6 +708,108 @@ public int Main(string str)
             context.WalkWithExitBlocks(7, 3);
         }
 
+        [TestMethod]
+        [TestCategory("Symbolic execution")]
+        public void ExplodedGraph_SwitchExpression_AllPatterns()
```

Comment:
All constant patterns? 😄

Reply:
Nope, this actually tests all kind of patterns:
`int i` is `VariableDeclarationPattern`,
`{ }` is (empty) `RecursivePattern`,
`_` is `DiscardPattern`